tests/test_internals/test_si.py

```diff
@@ -577,6 +576,23 @@ def test_group_set_custom_description(self):
                 else:
                     assert nwb.electrodes["group_name"][i] == "M1"
                     assert nwb.electrodes["group"][i].description == "M1 description"
+                    
+    def test_non_ints_as_channel_ids(self):
+        channel_ids = self.RX.get_channel_ids()
+        renamed_channel_ids = [str(id) for id in channel_ids]
+        self.RX3 = se.subrecordingextractor.SubRecordingExtractor(
+            parent_recording=self.RX, channel_ids=channel_ids, renamed_channel_ids=renamed_channel_ids,
```

Comment:
@h-mayorquin note that non-int channels are not formally supported by the old SI API. I think it would be great to substitute the extractor with a new-API extractor for the test.

Reply:
Ok. It should be like this now.